vaadin-spreadsheet-flow-parent/vaadin-spreadsheet-testbench/src/main/java/com/vaadin/flow/component/spreadsheet/testbench/SpreadsheetElement.java

```diff
@@ -24,6 +21,20 @@
 @Element("vaadin-spreadsheet")
 public class SpreadsheetElement extends TestBenchElement {
 
+    private TestBenchElement getSpreadsheetInShadowRoot() {
+        return this.$(TestBenchElement.class).all().stream()
+                .filter(el -> el.hasClassName("v-spreadsheet")).findFirst()
+                .get();
+    }
```

Comment:
I guess this can return `null` in cases, e.g., the Spreadsheet instance hasn't been attached to a page, so the other methods calling it should probably test for it, otherwise, an `NPE` can be thrown.

Reply:
Done